### ISSUE TITLE
Struggling with getting started

### ISSUE THREAD

**Issue by dowdian** [nvisionative/nvQuickTheme#187]
First, let me say Happy Thanksgiving!

## Please summarize your question in one sentence
**URGENT** 
`Yarn Install` command fails with pages and pages of errors. 

## Give a more extended description
I am trying desperately to get this working. I have read your documentation and watched your video. I have found that my experience diverges from that in your video and I don't know what to do to resolve the errors I'm getting. The error text in its entirety is included in the "Path to Error" text file below.

## Steps to reproduce (if needed)
I have included all the steps leading to the error in the "Path to Error" text file below, including the versions of the tools I'm using. I'm running a fairly new instance of Windows 10 64bit. I'm using the Command Prompt as Administrator to run these commands.

## Other comments or remarks
At the end of the "Path to Error" text file, you will see that I attempted to run `Gulp Package` after the yarn failure, just to see what happens, in case there are any clues there for you.

[Path to Error.txt](https://github.com/nvisionative/nvQuickTheme/files/3900652/Path.to.Error.txt)

**Comment by david-poindexter** [comment]
@dowdian Happy Thanksgiving to you!

Thanks for the details on the issues you are having.  The first thing I notice is you are using Node 12.  I honestly haven’t tested on Node 12, but I can confirm it works on Node 10.

There are some other things I notice (e.g., version of Gulp CLI should be 4) but let’s get Node sorted first.  If you do not have Node Version Manager (nvm) installed, I highly recommend using that to easily switch your active version of Node.  `nvm list` will list all installed versions.  `nvm use <version>` will change to the desired version.

Let us know what happens when you are running on Node 10.

**Comment by dowdian** [comment]
Good morning sir. I hope your Thanksgiving was as relaxing and delicious as ours was.
I have installed Cory Butler's NVM for Windows and used it to install Node 10.16.3. I followed up by installing yarn and gulp for this version of node. After this, I was able to successfully run the Yarn Install command, as well as all the gulp commands described in your documentation. I've now installed the new theme in a DNN instance and everything is as it should be. I've put the results of my work in the attached text file.

Thank you so much for your time and effort on this.

[Second Attempt.txt](https://github.com/nvisionative/nvQuickTheme/files/3911419/Second.Attempt.txt)
